### ISSUE TITLE
Bitmaps do not show up in the TI Script-Editor

### ISSUE THREAD

**Issue by johannes-wolf** [ndless-nspire/Luna#14]
BMP-Files added using Luna do not show up in the image tab of the TI-Nspire Script-Editor (Student Software).

I have added a bmp and a lua file using `lua test.bmp test.lua`. The lua file is correctly added to the tns, but the bmp does not show up.

**Comment by Vogtinator** [comment]
Yeah, that's a bit more complicated to do. The images need to be referenced by the problem XML document as `img_info` elements and a specially formatted `_RES` metadata item needs to be added to assign resource names to the image. It might be enough to add the `img_info` tags for making the script editor aware of it.

The Ndless installer for OS 3.9 is an example on how such a document looks: https://github.com/ndless-nspire/Ndless/blob/7ce6b732a6224ed9c017d1e9c36bb2429c947dbe/ndless/src/installer-3.9/Problem1_pre.xml

**Comment by Alekuso** [comment]
I'm a bit confused about the formatting of the `_RES` metadata. At some part of it, I can't really find a relation between the bmp and the res metadata. I need some help to see how I should name it depending on my images

**Comment by Vogtinator** [comment]
That's a custom format of a string list: Number of strings followed by length of string, then the string itself.
The numbers are encoded as three capital letters from A(0)-P(15), which can be seen as alternate encoding for hex (0x0 - 0xF). For the value in the 3.9 installer this results in:

`AAE` -> 0x004 -> Four strings

`ACE` -> 0x024 -> 36 characters

`1C86D1B39DEB85D42F438DE2C0C3BF78.BMP` the string

`AAG` -> 0x006 -> 6 characters

`device` the string

`ACE` -> 0x024 -> 36 characters

`BBF7096DCAF1CD03DE8364E14DB58939.BMP` the string

`AAE` -> 0x004 -> 4 characters

`plug` the string

This results in `1C86D1B39DEB85D42F438DE2C0C3BF78.BMP` using `device` and `BBF7096DCAF1CD03DE8364E14DB58939.BMP` using `plug` as resource name.

**Comment by Alekuso** [comment]
Thanks for helping me out. Everything is detected as it should, however I think I have some problem with my bitmaps, I don't really know if I converted it using the right format or anything as I get these kinds of results.
![image](https://user-images.githubusercontent.com/48645888/235898800-23f13e3e-2c74-4647-8f29-8738bba73031.png)

When using the one on the ndless installer, it works fine.
From what I see, the image viewer can't open the ndless bmp one but for some reasons it can open mines.

I wanted to see the hex data directly and I can't really find out why I don't get the "right" bmp
![image](https://user-images.githubusercontent.com/48645888/235899419-50c69b6e-7db2-4c78-a234-521d67ed331a.png)
![image](https://user-images.githubusercontent.com/48645888/235899526-d5c40d5f-a584-4bd1-84c3-c4d51ddbaded.png)

I tried to use multiple converters, however none of them work.

**Comment by Vogtinator** [comment]
The images have a specific BMP format. You can check https://github.com/ndless-nspire/Ndless/tree/master/ndless-sdk/tools/bmputil
